app/src/docs/v1.api-spec.yaml

```diff
@@ -231,35 +362,77 @@ components:
             - fizz@gov.bc.ca
         context:
           type: object
-          description: A freeform JSON object of key-value pairs.  All keys must be alphanumeric or underscore.
+          description: >-
+            A freeform JSON object of key-value pairs.  All keys must be
+            alphanumeric or underscore.
           example:
             something:
               greeting: Hello
               target: World
             someone: user
-    EmailRequest:
+        delay:
```

Comment:
The caller should not be calculating a delay in ms, they should give us a UTC time for delivery.

Reply:
I've renamed and reannotated `delay` to `delayUTC` to make it more subtly obvious we're expecting a UTC time instead of a millisecond delay.

Reply:
Changed again to delayTS as per discussions.